playbooks/k8s-cluster/nfs-client-provisioner.yml

```diff
@@ -0,0 +1,41 @@
+---
+# This playbook is used to setup the nfs-client-provisioner
+# If the cluster already has an nfs server setup run with --skip-tags="nfs_server,nfs_mkdir"
+#   Otherwhise an NFS server will be configured on kube-master[0] 
+# After settting up an NFS server, nfs software is installed on all nodes
+# Lastly, the nfs-client-provisioner is helm installed using the role
+#
+# Install the required NFS software on all nodes and then helm install the nfs-client 
+- hosts: kube-master[0]
+  tasks:
+  - name: Make sure NFS directory exists and is globally writable
+    file:
+      path: "{{ k8s_nfs_export_path }}"
+      state: directory
+      mode: "u=rwx,g=rwx,o=rwx"
+  tags:
+    - nfs_mkdir
+
+- hosts: kube-master[0]
+  become: yes
+  roles:
+    - { role: nfs, nfs_is_server: yes }
```

Comment:
I'm sure if these NFS steps should be in this standalone playbook or if it should be in k8s-cluster.yml and make a direct call to the nfs-server.yml. Any thoughts?